toolchain/parser/parse_node_kind.def

```diff
@@ -9,78 +9,99 @@
 // desired output. The x-macro for this header is `CARBON_PARSE_NODE_KIND`. The
 // definition provided will be removed at the end of this file to clean up.
 
-#ifndef CARBON_PARSE_NODE_KIND
-#error "Must define the x-macro to use this file."
+#if !(defined(CARBON_PARSE_NODE_KIND) ||          \
+      (defined(CARBON_PARSE_NODE_KIND_BRACKET) && \
+       defined(CARBON_PARSE_NODE_KIND_CHILD_COUNT)))
+#error "Must define the CARBON_PARSE_NODE_KIND x-macro to use this file."
```

Comment:
This error doesn't seem accurate any more?

Reply:
Adjusted message, does the rephrasing work for you? I'm trying to keep it short to fit on one line for #error